rclrs/Cargo.toml

```diff
@@ -43,10 +43,9 @@ tempfile = "3.3.0"
 tokio = { version = "1", features = ["rt", "time", "macros"] }
 
 [build-dependencies]
-# Needed for FFI
-bindgen = "0.70"
```

Comment:
Is there a reason `bindgen` was removed?

Reply:
It's not used anymore, the bindings are generated via the `generate_binding.py` script, which invokes the `bindgen` binary.